definitions/procedures/restore/installer_reset.rb

```diff
@@ -12,28 +12,12 @@ def run
 
     def installer_cmd
       installer = "yes | #{feature(:installer).installer_command} "
```

Comment:
someone needs to remind me, why this `yes |` is required, but out of scope for this PR probably.